src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -82,11 +83,13 @@ def getNodeShape(self, preemptable=False):
     def sshLeader(cls, clusterName):
         leader = cls._getLeader(clusterName)
         logger.info('SSH ready')
-        cls._sshAppliance(leader.ip_address, 'bash')
+        tty = sys.stdin.isatty()
+        cls._sshAppliance(leader.ip_address, 'bash', tty=tty)
 
     @classmethod
-    def _sshAppliance(cls, leaderIP, command):
-        command = "ssh -o \"StrictHostKeyChecking=no\" -t core@%s \"docker exec -i leader %s\"" % (leaderIP, command)
+    def _sshAppliance(cls, leaderIP, command, tty=False):
+        ttyFlag = 't' if tty else ''
+        command = "ssh -o \"StrictHostKeyChecking=no\" -t core@%s \"docker exec -i%s leader %s\"" % (leaderIP, ttyFlag, command)
```

Comment:
Change outer string to single quotes and eliminate backslashes before inner double quotes.

Think of a backslash as a character to avoid. Sorry for pushing back this PR for such a small thing but I have been asking people to avoid backslashes for about a year now. This should not be news.